documentation/library_guide/api_for_sycl_kernels/random.rst

```diff
@@ -107,6 +107,9 @@ Distribution                   Description
 ``cauchy_distribution``        Produces real values according to the Cauchy distribution
 ============================== ============================================================================
 
+.. note::
+  ``bernoulli_distribution``, ``geometric_distribution``, and ``uniform_int_distribution`` can only be used on devices with FP64 support as they rely on double precision in their implementation.
```

Comment:
Should it say "... as their implementation requires double precision"? As far as I can say, this is certainly true for the first two distributions.

By the way, can the implementation of `uniform_int_distribution` be modified to use FP32 in case FP64 is not available?

Reply:
Thanks, no objections.